### ISSUE TITLE
SA1000 report violation on spaces for C# 9.0 Target-Typed new expression

### ISSUE THREAD

**Issue by Debbus72** [DotNetAnalyzers/StyleCopAnalyzers#3297]
I'm running the Visual Studio 2019 preview (16.9.0-preview 3.0) and I'm getting a warning for the new Target-Typed new expression: [proposal](https://github.com/dotnet/csharplang/blob/master/proposals/csharp-9.0/target-typed-new.md). I'm not sure what the rules are for upcoming features that are still proposals. So please ignore me if I'm too fast.

Giving the following C# syntax:
```C#
// C# 9.0 - Target-Typed new expression
System.Guid value = new(bytearray);

// pre-C# 9.0
System.Guid value = new System.Guid(bytearray);
var value = new System.Guid(bytearray);
```

The new C# 9.0 Target-Typed new expression triggers a SA1000 warning "The keyword 'new' should be followed by a space." which in this case would be a false-positive.

**Comment by sharwell** [comment]
Duplicate of #3214
